templates/credentials/index.html

```diff
@@ -36,6 +36,13 @@ <h1>Canonical Ubuntu Essentials</h1>
             <a href="/credentials/shop" class="p-button--positive">Get the exam</a>
           </p>
         </div>
+        <div class="col-6 u-vertically-center u-align--center u-hide--medium u-hide--small">
+          <img id="takeover-image"
+               width="400"
```

Comment:
The image looks a bit way too big to me. but might be just me?
Also, optional but use image template?